DigitalLearningSolutions.Web.Tests/Controllers/Register/RegisterDelegateByCentreControllerTests.cs

```diff
@@ -213,36 +213,23 @@ public void LearnerInformationPost_updates_tempdata_correctly()
         }

```

Comment:
Looks like System.Collections.Generic and DigitalLearningSolutions.Data.Models.User are unused imports in this file

Reply:
Fixed